### PR TITLE
COMMERCE-4582 If item is plain string, do not assume it is an object

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
The issue is caused by this line 232 of `general.jsp`

As soon as the page refreshes after the form submission (upon "Save") the input of the Autocomplete component gets populated directly by the JSP, already localized, but the component assumes to find an object to take care of the localization itself. This is because the POST/GET flows are split between the Autocomplete component via headless and SSR via JSP. We should probably unify this flow, but for the time being... @marco-leo

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-channel-web/src/main/resources/META-INF/resources/channel/general.jsp [hunk 1]
```diff
@@ -243,5 +243,19 @@ if (shippingTaxCategory != null) {
 			'<%= liferayPortletResponse.getNamespace() %>shippingTaxSettings--taxCategoryId--',
 		itemsKey: 'id',
 		itemsLabel: ['name', 'LANG'],
+		onValueUpdated: function (value, shippingTaxData) {
+			console.log(arguments);
```
Comment (gianmarcobrunialti) [inline modules/apps/commerce/commerce-channel-web/src/main/resources/META-INF/resources/channel/general.jsp:247]: I still need to SF this. I know. 👍

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**